### PR TITLE
Fixing masters,workers hostnames and adding steps for OCP GA install

### PR DESCRIPTION
**Pull request by rlopez133**
# Description

Went ahead did the following for the install-steps guide:

- fixed typo in masters,worker naming 
- added steps on how to use OCP…GA release version

## Type of change

Please select the appropiate options:

- [ X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ X] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: install-steps.md [hunk 1]
```diff
@@ -511,6 +511,35 @@ sudo cp ./oc /usr/local/bin/oc
...
 
+## Retrieving the OpenShift Installer (GA Release)
+
+The `latest-4.x` (i.e. `latest-4.3`) may be used to deploy the latest 
+Generally Available version of Red Hat OpenShift Platform.
+
+Configure VARS
+    ~~~sh
+    export VERSION=latest-4.3
+    export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs)
```
Comment (e-minguez) [inline install-steps.md]: No need for grep or xargs:
```
curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | awk '/Pull From: quay.io/ { print $3 }'
```

File: install-steps.md [hunk 2]
```diff
@@ -511,6 +511,35 @@ sudo cp ./oc /usr/local/bin/oc
...
+
+Once the installer has been chosen, the next step is to extract it. 
+
+~~~sh
+export cmd=openshift-baremetal-install
+export pullsecret_file=~/pull-secret.txt
+export extract_dir=$(pwd)
+# Get the oc binary
+curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/openshift-client-linux-$VERSION.tar.gz | tar zxvf - oc
+sudo cp ./oc /usr/local/bin/oc
```
Comment (e-minguez) [inline install-steps.md]: I'd say to use `sudo mv` to avoid having the binary messing around

File: install-steps.md [hunk 3]
```diff
@@ -488,15 +488,15 @@ Two approaches:
     export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs)
```
Comment (e-minguez) [inline install-steps.md]: No need for grep or xargs:
```
curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/release.txt | awk '/Pull From: quay.io/ { print $3 }'
```

File: install-steps.md (hunk 1 above)
Comment (sreichar) [reply install-steps.md]: xargs with no args is and echo - basically this is doing white space removal.

File: install-steps.md (hunk 3 above)
Comment (sreichar) [reply install-steps.md]: see above

File: install-steps.md (hunk 1 above)
Comment (e-minguez) [reply install-steps.md]: I think it still not needed:

```
$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.3/release.txt | awk '/Pull From: quay.io/ { print $3 }'
quay.io/openshift-release-dev/ocp-release@sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d
```
```
$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest-4.3/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs
quay.io/openshift-release-dev/ocp-release@sha256:3a516480dfd68e0f87f702b4d7bdd6f6a0acfdac5cd2e9767b838ceede34d70d
```

Comment (rlopez133) [reply install-steps.md]: Can we just leave as is? I don't see harm in leaving it in the expression.

File: install-steps.md (hunk 2 above)
Comment (rlopez133) [reply install-steps.md]: Not sure what you mean here. Currently Ansible using the copy module to the above and works fine.

File: install-steps.md (hunk 3 above)
Comment (rlopez133) [reply install-steps.md]: I'm going to say this is doesn't matter and leaving the expression doesn't hurt considering we have validated with it.

**Pull request merged**